src/clib/pio.h

```diff
@@ -595,7 +653,7 @@ extern "C" {
                             MPI_Comm io_comm, int *iosysidp);
     int PIOc_get_numiotasks(int iosysid, int *numiotasks);
     int PIOc_Init_Intracomm(MPI_Comm comp_comm, int num_iotasks, int stride, int base, int rearr,
-                            int *iosysidp);
+                            rearr_opt_t *rearr_opts, int *iosysidp);
```

Comment:
I think that Ed is going to argue that we do not want derived types as part of our PIOc interface.   
Can we maintain the derived type in fortran but list all 6 opts individually in the PIOc functions?